subsys/suit/platform/src/suit_plat_write.c

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/logging/log.h>
+#include <stdbool.h>
+#include <suit_platform.h>
+#include <suit_plat_decode_util.h>
+#include <suit_plat_error_convert.h>
+#include <suit_platform_internal.h>
+#include <suit_plat_write_domain_specific.h>
+#ifdef CONFIG_SUIT_MANIFEST_VARIABLES
+#include <suit_manifest_variables.h>
+#endif /* CONFIG_SUIT_MANIFEST_VARIABLES */
+
+LOG_MODULE_REGISTER(suit_plat_write, CONFIG_SUIT_LOG_LEVEL);
+
+#ifdef CONFIG_SUIT_MANIFEST_VARIABLES
+static int suit_plat_check_write_mfst_var(suit_component_t dst_handle, struct zcbor_string *content,
+					  struct zcbor_string *manifest_component_id,
+					  struct suit_encryption_info *enc_info)
+{
+	struct zcbor_string *component_id = NULL;
+	suit_plat_err_t plat_ret;
+	uint32_t id;
+	uint32_t val;
+	int ret;
+
+	if (enc_info != NULL) {
+		LOG_ERR("Encryption not supported for MFST_VAR components");
```

Comment:
Again - I do not thin an error is necessary here, a warning would be sufficient

Reply:
Agreed to leave error level here.